packages/vats/package.json

```diff
@@ -32,6 +32,7 @@
     "@agoric/install-ses": "^0.5.9",
     "@agoric/marshal": "^0.4.7",
     "@agoric/nat": "^4.0.0",
+    "@agoric/notifier": "*",
```

Comment:
Please get the latest `@agoric/notifier` version and use `"^NOTIFIER-VERSION"` instead of `"*"`:

```sh
grep '"version":' packages/notifier/package.json
```

Reply:
done